packages/orchestration/src/contracts/stakingAccountHolder.js

```diff
@@ -41,6 +47,8 @@ const HolderI = M.interface('holder', {
   makeCloseAccountInvitation: M.call().returns(M.promise()),
   makeTransferAccountInvitation: M.call().returns(M.promise()),
   delegate: M.callWhen(M.string(), AmountShape).returns(M.string()),
+  queryBalance: M.callWhen().optional(M.string()).returns(M.any()),
```

Comment:
we have a better type than `any`. maybe we can get codegen to make shapes. until then, I think it's worth hand rolling